### ISSUE TITLE
A new developer reply to a review does not load after deletion

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#6447]
### Describe the problem and steps to reproduce it:

- Sign in as a developer
- Go to one of your add-ons
- Go to the review listing
- Reply to a review
- Click on the timestamp 
- Delete the reply
- Reply to the review again
- Click on the timestamp again

### What happened?

Nothing happens

### What did you expect to happen?

You should see the permalink screen of the review reply

### Anything else we should know?

If you refresh the page, you can see the permalink screen as expected.

**Comment by xu3u4** [comment]
Hi @kumar303,
I found it is an interesting bug. I wish I can fix it.
But there is something I am not sure...
Please see the PR for the details 🙂

**Comment by kumar303** [comment]
Yes, this one is interesting. It is a hard one to fix so I think we should not spend time on fixing it right now. Thanks again for your time on it. I left a comment on the PR: https://github.com/mozilla/addons-frontend/pull/8007#pullrequestreview-239578153

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-158
